thrall/app/lib/ThrallConfig.scala

```diff
@@ -25,4 +25,5 @@ class ThrallConfig(override val configuration: Configuration) extends CommonConf
 
   lazy val from: Option[DateTime] = properties.get("rewind.from").map(ISODateTimeFormat.dateTime.parseDateTime)
 
+  lazy val bulkIndexBucket: String = properties("s3.bulk.index.bucket")
```

Comment:
Thrall now has a new config to read from `/etc/gu/thrall.properties`.

Reply:
do we need that ? s3Path in message should be full